### DISCUSSION TITLE
Biggest Hurdles in Freelance Web-Development

### DISCUSSION THREAD

**Discussion by vortand** [EddieHubCommunity/support#619] [Q&A]
Hi Everyone, I'm from Central Europe and I would like to get into freelance website development, front-end and back-end, but I would like to learn more about the industry and what it looks like.

If some hardcore freelancers could answer these questions, I would be super grateful!

1. What is the most common order you receive and what is the most profitable order you receive (e.g. Wordpress site or old website upgrade)
2. How much do you charge on average per order?
3. What are the biggest hurdles/problems in your experience in this industry?
4. How do you manage to find clients consistently?

If anyone would be willing to, I would also be happy to have a short call with you to discuss about the industry **in exchange for some social media marketing tips** for your freelancing business (I have a few years of experience in this sector)

Thanks a lot,

David

**Comment by eddiejaoude** [comment]
Great question @vortand 👍 

From experience here are some tips:

- Portfolio website to showcase your previous work
- Open source contributions to showcase to bigger projects who want to see your code / collaboration / communication etc
- Share your journey on socials, it is a great way for people to find you (have your name on all socials be the same and ideally your actual name)
- I know platforms like UpWork are sometimes and initially a race to the bottom, but as your services / skills become more in demand and you have good testimonials, you can then start to increase your rate / pricing 
- Network on socials and open source (which is social coding) is great to meet other techies, not just to learn from but also when they get busy they can refer work your way

Freelancing is the best decision I ever made (over 10 years ago). It is not easy, it is hard work especially at the beginning, but highly rewarding. 

All the best and I look forward to hearing how it goes and also other peoples suggestions and opinions 🤓
